src/test/java/picard/cmdline/PicardCommandLineTest.java

```diff
@@ -63,4 +63,19 @@ public void testLaunchAllCommandLineProgramsWithBarclayParser() {
     public void testPrintUsage() {
         Assert.assertEquals(new PicardCommandLine().instanceMain(new String[]{"-h"}), 1);
     }
+
+    @Test
+    public void testCommandlineProgramPropertiesOneLineSummaryLength() {
+        // Find and test each command line tool to ensure that one line
+        // summaries don't exceed the maximum allowable length
+        PicardCommandLine.processAllCommandLinePrograms(
+                Collections.singletonList("picard"),
```

Comment:
It's not too bad, but it does seem wasteful that the tests generates the list of all picard CLPs three times here. If you generated it once in a `BeforeClass` you could use it in all three tests.

Reply:
Done. Consolidated these into a single `BeforeClass` discovery method.